nautobot_golden_config/templates/nautobot_golden_config/compliance_device_report.html

```diff
@@ -107,30 +107,7 @@ <h1>{% block title %}Configuration Compliance - {{ device.name }}{% endblock %}<
             {% endif %}
 
         </tr>
-        {% if item.compliance and not item.rule.config_ordered %}
-            <tr>
-                <td style="width:250px">Intended Configuration</td>
-                <td class="config_hover">
-                    <span id="{{ item.rule|slugify }}_intended"><pre>{{ item.intended|placeholder }}</pre></span>
-                    <span class="config_hover_button">
-                        <button class="btn btn-inline btn-default hover_copy_button" data-clipboard-target="#{{ item.rule|slugify }}_intended">
-                            <span class="mdi mdi-content-copy"></span>
-                        </button>
-                    </span>
-                </td>
-            </tr>
-            <tr>
-                <td style="width:250px">Actual Configuration</td>
-                <td class="config_hover">
-                    <span id="{{ item.rule|slugify }}_actual"><pre>{{ item.actual|placeholder }}</pre></span>
-                    <span class="config_hover_button">
-                        <button class="btn btn-inline btn-default hover_copy_button" data-clipboard-target="#{{ item.rule|slugify }}_actual">
-                            <span class="mdi mdi-content-copy"></span>
-                        </button>
-                    </span>
-                </td>
-            </tr>
-        {% elif item.compliance %}
+        {% if item.ordered %}
```

Comment:
I don't think so :)

That was the rule, but not sure it matters. I will document this in the PR, but from an internal thread realized:

Use Cases:
* Ordering Matters and compliant - Only show configuration (not intended or actual), since it the exact same thing
* Non-Compliant - Show intended and actual
    * Ordering Matters and it is not the same == Non-compliant, nothing special needs to be done
* Compliant, but ordering does not match - Show intended and actual

So, given the use cases, there are only two results:
* exact match, only show "configuration"
* Other, show but intended and actual

I do not know if those code works, but conceptually, that was my thought process. 


Looking at the code, I believe that netutils returns a dictionary with a key of `ordered_compliant`, which get's saved to the model's `.ordered` method.

Reply:
Just realizing how this came off, `I don't think so :)`, I meant it as "I don't think so, but I am confused myself, so I could be wrong" 

I did not mean anything negative, more calling myself out for not knowing.